lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -62,13 +62,32 @@
     aliases:
       - incremental
 
+  enable_ipv6:
+    version_added: 2.8
+    description:
+      - Enable IPv6 networking.
+    type: bool
+    default: null
+    required: false
+
   ipam_driver:
     description:
       - Specify an IPAM driver.
 
   ipam_options:
     description:
       - Dictionary of IPAM options.
+      - Deprecated in 2.8, will be removed in 2.12. Use parameter ``ipam_config`` instead. In Docker 1.10.0, IPAM
+        options were introduced (see L(here,https://github.com/moby/moby/pull/17316)). This module parameter addresses
+        the IPAM config not the newly introduced IPAM options.
+
+  ipam_config:
+    version_added: 2.8
+    description:
+      - List of IPAM config blocks. Consult docker docs for valid options and values.
+    type: list
+    default: []
```

Comment:
Most of the other options (e.g. `connected`) also have an empty list as default so I di the same. Changing to `None`.

Reply:
`connected` is a bit different, since it is not an option passed to docker, but something processed completely by this module. I would have not used defaults for some of the others, but that's now too late to change (to keep backwards compatibility).